plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py

```diff
@@ -108,22 +108,7 @@
 chdir_plugin_root()
 
 
-def build_ray_launcher_wheel(tmpdir: str) -> str:
-    """
-    This  only works on ray launcher plugin wheels for now, reasons being in our base AMI
-    we do not necessarily have the dependency for other plugins.
-    """
-    command = "python -m pip --disable-pip-version-check list | grep hydra | grep -v hydra-core "
-    output = subprocess.getoutput(command).split("\n")
-    plugins_path = [x.split()[0].replace("-", "_") for x in output]
-    assert (
-        len(plugins_path) == 1 and "hydra_ray_launcher" == plugins_path[0]
-    ), "Ray test AMI doesn't have dependency installed for other plugins."
-
-    return build_plugin_wheel(tmpdir)
-
-
-def build_plugin_wheel(tmp_wheel_dir: str) -> str:
+def build_ray_launcher_wheel(tmp_wheel_dir: str) -> str:
     chdir_hydra_root()
     plugin = "hydra_ray_launcher"
     os.chdir(Path("plugins") / plugin)
```

Comment:
You can probably use monkeypatch to change the directory and restore automatically.

Reply:
I didn't know `monkeypatch` could do that, I thought we usually do `fixture`?

Reply:
https://github.com/facebookresearch/hydra/blob/master/tests/test_hydra.py#L1049-L1050

Reply:
I see, thanks!